client/my-sites/site-settings/section-import.jsx

```diff
@@ -94,22 +96,20 @@ class SiteSettingsImport extends Component {
 			}
 		);
 
-		if ( isJetpack ) {
-			return (
-				<EmptyContent
+		return (
+			<Main>
+				<HeaderCake backHref={ '/settings/general/' + siteSlug }>
+					<h1>{ translate( 'Import' ) }</h1>
+				</HeaderCake>
+				{ isJetpack && <EmptyContent
 					illustration="/calypso/images/drake/drake-jetpack.svg"
 					title={ translate( 'Want to import into your site?' ) }
 					line={ translate( 'Visit your site\'s wp-admin for all your import and export needs.' ) }
 					action={ translate( 'Import into %(title)s', { args: { title } } ) }
 					actionURL={ adminUrl + 'import.php' }
 					actionTarget="_blank"
-				/>
-			);
-		}
-
-		return (
-			<div className="section-import">
-				<EmailVerificationGate>
+				/> }
+				{ ! isJetpack && <EmailVerificationGate>
```

Comment:
Just to follow up about my comment above about changing `! isJetpack` to `isJetpack === false`: this will not be needed here, as there's a `if ( ! site ) { return null }` above.